delta/sdk-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/views/model/IndexingMessage.scala

```diff
@@ -0,0 +1,122 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.views.model
+
+import cats.syntax.all._
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfError
+import ch.epfl.bluebrain.nexus.delta.rdf.Triple.subject
+import ch.epfl.bluebrain.nexus.delta.rdf.graph.Graph
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.api.{JsonLdApi, JsonLdJavaApi}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.JsonLdContext.keywords
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.RemoteContextResolution
+import ch.epfl.bluebrain.nexus.delta.sdk.EventExchange
+import ch.epfl.bluebrain.nexus.delta.sdk.EventExchange.{EventExchangeResult, EventExchangeValue}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{BaseUri, ResourceRef}
+import ch.epfl.bluebrain.nexus.delta.sdk.syntax._
+import io.circe.Json
+import monix.bio.IO
+import org.apache.jena.graph.Node
+
+sealed trait IndexingMessage {
```

Comment:
We already use the term `Message`, maybe something like `ViewData` ?

Reply:
`ViewData` is probably better, but also a bit confusing, because it's not always data, sometimes it's not found :)